### ISSUE TITLE
MAV_CMD_REQUEST_VIDEO_STREAM_INFORMATION reply not implemented

### ISSUE THREAD

**Issue by profff** [Dronecode/camera-manager#198]
is there any reason why MAV_CMD_REQUEST_VIDEO_STREAM_INFORMATION request is not implemented ? 
I can try to make a pull request but as I spend too mutch time ton configure and understand working and issue I dont know If my boss will be ok for that ! :D

**Comment by hamishwillee** [comment]
Yes, the message was added to MAVLink after Intel stopped providing resource to support this library. If you provide a PR I am sure @lbegani will review/merge it though.
